package.json

```diff
@@ -83,7 +83,7 @@
     "madge": "3.3.0",
     "mocha": "5.1.1",
     "mock-require": "3.0.2",
-    "nyc": "11.7.3",
+    "nyc": "13.1.0",
```

Comment:
updated this dep to be consistent with our other repos and it is a no-risk change